clang/lib/Driver/Driver.cpp

```diff
@@ -1086,8 +1098,20 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
               << SYCLForceTarget->getAsString(C.getInputArgs());
 
         for (StringRef Val : SYCLTargetsValues->getValues()) {
+          StringRef Val_t(Val);
+          // Handle target specifications that resemble 'intel_gpu_*' here.
```

Comment:
Great to have comments at least!
To move to the `isIntelGPUTarget` function?
same for the following same comments?